cloud/keystone_test.go

```diff
@@ -368,19 +373,52 @@ var _ = Describe("Keystone client", func() {
 			})
 
 			It("Should read tokens scoped to admin project", func() {
-				server.AppendHandlers(
-					ghttp.RespondWithJSONEncoded(201, getV3TokensAdminResponse()),
-					ghttp.RespondWithJSONEncoded(200, getV3TokensAdminResponse()),
-				)
+				setResponses(getV3TokensAdminResponse())
 				setupV3Client()
+
 				auth, err := client.VerifyToken(token)
+
 				Expect(err).To(BeNil())
 				Expect(auth.TenantID()).To(Equal("admin-project-id"))
 				Expect(auth.TenantName()).To(Equal("admin-project"))
 				Expect(auth.DomainID()).To(Equal("default"))
 				Expect(auth.DomainName()).To(Equal("default"))
 				Expect(auth.IsAdmin()).To(BeTrue())
 			})
+
+			It("Should reject admin token with multiple roles", func() {
```

Comment:
nit: the following 2 tests can be written as `DescribeTable`

Reply:
I actually agree., especially since I added a third one :) Done.